rollup/conf/config.json

```diff
@@ -19,7 +19,10 @@
         "min_gas_price": 0,
         "gas_price_diff": 50000,
         "l1_base_fee_weight": 0.132,
-        "l1_blob_base_fee_weight":  0.145
+        "l1_blob_base_fee_weight":  0.145,
+        "commit_batch_timeout_window_minutes": 5,
```

Comment:
can we change this to another name? This name seems to indicate the timeout to commit a batch (i.e. we should commit batch every `commit_batch_timeout_window_minutes`).